tests/binary/binaries_test.go

```diff
@@ -106,9 +106,9 @@ func TestBinarySizes(t *testing.T) {
 	}{
 		// TODO: shrink the ranges here once the active work to reduce binary size is complete
 		// For now, having two small a range will result in lots of "merge conflicts"
-		"istioctl":        {60, 95},
+		"istioctl":        {60, 100},
 		"pilot-agent":     {20, 26},
-		"pilot-discovery": {60, 112}, // TODO: shrink this once we merge the multicluster ambient controller as default
+		"pilot-discovery": {60, 120}, // TODO: shrink this once we merge the multicluster ambient controller as default
```

Comment:
Not on you, but I've seen this number grow a couple of times in PRs this release. We should plan to do some work to drop this down (though, as the comment says, unifying the ambient multicluster stuff and maybe even centralizing the remote secrets logic could help)

Reply:
We can also get rid of bash/zsh autocomplete